src/main/java/tutorease/address/logic/commands/EditContactCommand.java

```diff
@@ -148,6 +148,24 @@ public String toString() {
                 .toString();
     }
 
+    /**
+    * Checks if the edited person would create a duplicate entry in the model based on unique attributes.
+    * This method verifies that the edited person's details do not conflict with existing persons in the model.
+    */
```

Comment:
javadocs here can follow the format mentioned in the textbook with appropriate tags like @params @throws

Reply:
Fixed!